test/language/line-terminators/S7.3_A3.4_T1.js

```diff
@@ -5,8 +5,7 @@
 info: Single line comments can not contain PARAGRAPH SEPARATOR (U+2029) inside
 es5id: 7.3_A3.4_T1
 description: Insert PARAGRAPH SEPARATOR (\u2029) into single line comment
-negative: ReferenceError
+negative: SyntaxError
 ---*/
 
-// CHECK#1
-eval("// single line \u2029 comment");
+// single line PS> ??? (invalid)
```

Comment:
It's not easy to tell this line contains a PS character, but still looks better than using eval.

Also, it seems the expected error should be the ReferenceError, which you can catch:

```js
try { // single line PS>  (invalid)
} catch(e) { console.log(e) }
```

I just removed the ??? characters, responsible for the SyntaxError. The ReferenceError occours because invalid is not defined, as the PS breaks the line.

Maybe:

```js
// single line PS>  throw new Test262Error()
```